### PR TITLE
Create index page with restaurants

### PR DESCRIPTION
**Pull request by mmollerstrom**
### Create index page with restaurants

https://www.pivotaltracker.com/story/show/145100541

* Created route for the home page
* Created a controller that had a hash with the restaurant names
* Created a view to display restaurant names

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,3 +1,7 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get 'home/index' 
+  root 'home#index'
```
Comment (tochman) [inline config/routes.rb]: Why 2 routes pointing to the same controller? 

I also prefer this syntax:
```
root, controller: :home, action: :index
```

File: features/index.feature [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Feature: Landing Page content
+  As a visitor
+  in order to access the app
+  I need to see an index page
+  I would like to be presented with some examples of restaurants that deliver food
```
Comment (tochman) [inline features/index.feature:5]: Is this the story we had in the Tracker. Feels weird.

**Review by tochman — CHANGES_REQUESTED** [review]

File: config/routes.rb (hunk 1 above)
Comment (mmollerstrom) [reply config/routes.rb]: Fixed!

File: features/index.feature (hunk 2 above)
Comment (mmollerstrom) [reply features/index.feature:5]: It was, but I just changed it to a more proper one

Comment (tochman) [reply features/index.feature:5]: I would suggest:
```
As a visitor
In order to get information about available restaurants
I would like to see a list of restaurants when I access the application
```

or something like that....

Comment (tochman) [reply features/index.feature:5]: Not a blocker though. Please update in Pivotal Tracker and read up on the User Story format.

**Review by tochman — APPROVED** [review]

**Pull request merged**